### PR TITLE
REF: exchange print statements for logger.debug

### PR DESCRIPTION
**Pull request by dchabot**

### CODE REVIEW

File: ophyd/controls/area_detector.py [hunk 1]
```diff
@@ -181,7 +184,7 @@ def configure(self, **kwargs):
         self._acquire.put(0)
         while self._acquire.value:
             time.sleep(0.5)
-            print('[!!] Waiting for camera to stop acquiring ....')
+            logger.debug'[!!] Waiting for camera to stop acquiring ....')
```
Comment (tacaswell) [inline ophyd/controls/area_detector.py]: These all lost the opening `(`

**Comment by klauer** [comment]
Despite how horribly annoying those `'waiting for ...'` messages can be, I'd rather see them than wonder why the scan is hanging. Would you agree with raising some of those to `logger.warning`?

**Comment by danielballan** [comment]
New area detector uses logging, not print.

**Comment by klauer** [comment]
Looking back on this issue, it may be nice to have some convenience functions to set logging levels on the appropriate logger for areaDetector (and related) debugging.

**Pull request closed without merging**